### ISSUE TITLE
Project abandoned? ( tl;dr nope )

### ISSUE THREAD

**Issue by danawoodman** [kristoferjoseph/flexboxgrid#248]
It appears no commits have occurred since Jan. Before I build off of flexboxgrid, I'd like to know if the project is abandoned?

**Comment by Oldenborg** [comment]
I feel the same way

**Comment by jdickey** [comment]
Several [forks](https://github.com/kristoferjoseph/flexboxgrid/network) seem to be under more active work; perhaps @kristoferjoseph would prefer to hand off to one of them if he's not interested in/able to maintain this further?

**Comment by ryanblock** [comment]
Or perhaps the software is presently in the state the maintainer intends it?

**Comment by SuEric** [comment]
also version 7.0 is actually older than master; way behind master... so yeah, looks like project is abandoned

**Comment by notapatch** [comment]
From Falieson:  https://github.com/kristoferjoseph/flexboxgrid/pull/211 and 28 days ago

> 
> @PolGuixe @b264 @hassanasad
> 
> I was able to get ahold of @kristoferjoseph and he has said
> 
> this repo is still being actively maintained
> all these addon features we want he believes are handled by @lgg 's https://github.com/lgg/flexboxgrid-helpers
> these features shouldn't be incorporated into flexboxgrid b/c he wants to keep the package size small
> Speak up if you have an issue with this resolution, I may rename my fork something like 'flexboxgrid-extended' if there is good reason to have a combined package but I think everyone should just be able to depend on the 2 packages.

Though it has been pointed out that the [pull request for the README about flexboxgrid-helpers][1] has not been accepted. 


[1]: https://github.com/kristoferjoseph/flexboxgrid/pull/246

**Comment by b264** [comment]
I read that also, but @kristoferjoseph did not have any github activity.

I've assumed it's unmaintained.

**Comment by kristoferjoseph** [comment]
I find these "issues" a bit silly to be honest.
The best thing a widely used open source project can be is stable.
Adding low impact features that add to css weight and possibly introduce bugs for existing users is a bad idea. 

I just got tired of repeating myself. 

If a legitimate​ bug is filed or a PR for a legitimate bug is filed I will gladly add it.

Thanks

**Comment by b264** [comment]
A readme update doesn't seem like it'd add any bugs or cause stability issues

**Comment by danawoodman** [comment]
@kristoferjoseph it concerns me when there are many unreplied open PRs/issues on a project with no updates for 6 months. If I'm going to rely on a project, I like to do my research first and make sure I'm depending on tools that will continue to be maintained. I think that is reasonable. 

That said, my intent isn't to attack, but to just learn if this project is planned on being maintained as many open source projects die (I know this first hand as I've killed a lot of my own projects).  It sounds like the intent to maintain exists, which answers my question. Closing

On a related note: If flexboxgrid-helpers is something people should be checking out, perhaps it could be added to the readme?
